### PR TITLE
Add 'hpt' xml to vmfeatures

### PR DESCRIPTION
**Pull request by chloerh**
Signed-off-by: haizhao <haizhao@redhat.com>

### CODE REVIEW

File: virttest/libvirt_xml/vm_xml.py [hunk 1]
```diff
@@ -2611,3 +2613,35 @@ def marshal_to_iothreads(tag, attr_dict, index, libvirtxml):
         if tag != 'iothread':
             return None
         return attr_dict['id']
+
+
+class VMFEATURESHPTXML(base.LibvirtXMLBase):
```
Comment (chunfuwen) [inline virttest/libvirt_xml/vm_xml.py]: class is all capital letter, I don't think it comply naming convention

File: virttest/libvirt_xml/vm_xml.py [hunk 2]
```diff
@@ -2611,3 +2613,35 @@ def marshal_to_iothreads(tag, attr_dict, index, libvirtxml):
...
+
+    """
+    Hpt tag XML class of features tag
+
+    Element:
+        resizing:               text
+        maxpagesize_unit:       attribute
+        maxpagesize:            int
```
Comment (chunfuwen) [inline virttest/libvirt_xml/vm_xml.py]: suggest add example xml here to make it more understandable

**Comment by chunfuwen** [comment]
@chloerh , I assume some previous case should be impacted by this change, please paste one running result

**Comment by chloerh** [comment]
type_specific.io-github-autotest-libvirt.papr_hpt.positive_test.disabled: PASS (49.80 s)

File: virttest/libvirt_xml/vm_xml.py (hunk 1 above)
Comment (chloerh) [reply virttest/libvirt_xml/vm_xml.py]: Updated

File: virttest/libvirt_xml/vm_xml.py (hunk 2 above)
Comment (chloerh) [reply virttest/libvirt_xml/vm_xml.py]: Updated although I don't see any other xml class has examples of their own in this file.

**Pull request merged**